src/pages/studyView/table/MultiSelectionTable.tsx

```diff
@@ -136,6 +142,11 @@ export class MultiSelectionTable extends React.Component<
         this.sortBy = this.props.defaultSortBy;
     }
 
+    @memoize
```

Comment:
nice, just want to confirm the memoization works since the argument is an object. is there a need to pass a memoization key?